spec/spec_helper.rb

```diff
@@ -80,6 +90,22 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.before type: :system do
+    # Disable UI system tests by default when testing locally
+    skip('UI system testing is currently disabled') unless ENV.fetch('ENABLE_UI_TESTING', nil) == 'true'
+
+    # Configure the selenium webdriver for system tests
+    browser = ENV.fetch('DISABLE_HEADLESS_UI_TESTING', nil) == 'true' ? :chrome : :headless_chrome
+    driven_by :selenium, using: browser, screen_size: [1400, 1400],
```

Comment:
This should be configured once globally, not in a `before` for each test. You should consult the Capybara documentation for how to do this.